metrics/system.go

```diff
@@ -0,0 +1,74 @@
+package metrics
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	metrics "github.com/rcrowley/go-metrics"
+	"github.com/shirou/gopsutil/process"
+)
+
+// CPUTime returns an amount of CPU time (in seconds) assigned to executor process
+// by system.
+func CPUTime() (float64, error) {
+	p := getExecutorProcess()
+	t, err := p.Times()
+	if err != nil {
+		return 0, fmt.Errorf("Unable to get CPU times: %s", err)
+	}
+
+	total := t.User + t.System + t.Nice + t.Iowait + t.Irq + t.Softirq +
+		t.Steal + t.Guest + t.GuestNice + t.Stolen
+
+	return total, nil
+}
+
+// Memory returns a number of bytes used by executor process.
+func Memory() (uint64, error) {
```

Comment:
We can use [`CaptureRuntimeMemStats`](https://godoc.org/github.com/rcrowley/go-metrics#CaptureRuntimeMemStats)  instead

Reply:
@janisz fixed